### DISCUSSION TITLE
How to jump to the main page after login

### DISCUSSION THREAD

**Discussion by mimimiaomiao1** [AvaloniaUI/Avalonia#8194] [Q&A]
In the prism, I do this
<img width="514" alt="image" src="https://user-images.githubusercontent.com/58240619/170297000-49c7f999-440c-4b16-b3d3-3a7736e829f5.png">

How can I modify it
<img width="560" alt="image" src="https://user-images.githubusercontent.com/58240619/170297149-8816f306-05de-46c5-8dcb-f468dfa97078.png">

**Comment by mimimiaomiao1** [comment]
I need help.

**Comment by maxkatz6** [comment]
It doesn't seem to be an Avalonia related question or issue. Please ask it in Prism.Avalonia repository.
